packages/core/src/sass/app/_common/layout/modal-root.scss

```diff
@@ -1,6 +1,6 @@
 .modal-root {
-    width: 100vw;
-    height: 100vh;
+    width: 100%;
+    height: 100%;
```

Comment:
@farhan-nurzi-deriv can you explain why we are making this change?

Reply:
I will move it to `tablet-landscape` css instead to avoid bug in other places